lib/responders.rb

```diff
@@ -1,11 +1,5 @@
-require 'action_controller'
```

Comment:
I finally got it. I restored `require 'action_controller'` and it's sufficient to solve my problem. Sorry and thanks for your comments.